js/youre_square.js

```diff
@@ -0,0 +1,7 @@
+// https://www.codewars.com/kata/youre-a-square/train/javascript
+
+var isSquare = function(n){
+  return (Number.isInteger(Math.sqrt(n)))
```

Comment:
Interesting - the semicolon at the end is missing. Moreover, the outer parentheses are unnecessary :smile: